xray/utils/resultstable.go

```diff
@@ -202,6 +258,13 @@ type cveRow struct {
 	cvssV3 string `col-name:"CVSS\nv3" extended:"true"`
 }
 
+type ResultsJsonTable struct {
```

Comment:
ResultsJsonTable --> ResultsSimpleJson

Let's also add a description for this struct, explaining its role.